torch/csrc/distributed/rpc/ScriptCall.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <c10/util/Optional.h>
+#include <torch/csrc/distributed/rpc/Message.h>
+#include <torch/csrc/jit/operator.h>
+#include <torch/csrc/jit/pickler.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+using torch::jit::Operator;
+
+// A ScriptCall instance represents an invocation of a builtin operator for a
+// TorchScript function (not implemented yet). If it is a builtin operator, it
+// contains a shared ptr to the `Operator` and a list of arguments.
+class TORCH_API ScriptCall final {
```

Comment:
Not sure why this is called `ScriptCall` and the message type is called `SCRIPT_CALL` too. IIUC, this doesn't really have anything to do with Torchscript. We are just calling a builtin operator. We should call this BuiltinOpCall or something.